lib/components/ListPicker.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <select :value="value" ref="listpicker" @change="onSelectedIndexChange($event)" @input="updateValue()">
+        <option v-if="disabledVal" disabled :value="disabledVal">{{ disabledVal }}</option>
+        <option v-for="item in items" :key="item[valueProp]" :value="item[valueProp]">
+                {{ item[labelProp] }}
+        </option>
+    </select>
+</template>
+
+<script>
+
+export default {
+    name: 'ListPicker',
+    props: [
+        'disabledVal', // Works only in web.
+        'items',
+        'valueProp',
+        'labelProp',
+        'selectedIndex',
+        'selectedIndexChange', // an event.
+        'value',
```

Comment:
not sure if we need value here either. It should be internal and driven through selectedIndex

Reply:
I agree. We do not need selectedIndex prop as we have it as value prop.

Reply:
v-model and selectedIndex should be both supported I think. We just don't need value prop in addition